runtime/runtime_controller.cc

```diff
@@ -233,14 +233,15 @@ bool RuntimeController::ReportTimings(std::vector<int64_t> timings) {
   return false;
 }
 
-bool RuntimeController::NotifyIdle(int64_t deadline) {
+bool RuntimeController::NotifyIdle(int64_t deadline, size_t freed_hint) {
   std::shared_ptr<DartIsolate> root_isolate = root_isolate_.lock();
   if (!root_isolate) {
     return false;
   }
 
   tonic::DartState::Scope scope(root_isolate);
 
+  Dart_HintFreed(freed_hint);
```

Comment:
If the ordering of these two calls is important, then a comment here explaining that will probably be helpful.

Reply:
Done.